rfcs/0001/Analysis-to-map-MarcXML-fields-to-CDSVideos.md

```diff
@@ -0,0 +1,1461 @@
+| Category/ Collection                              | ??                     | category                          | Yes       |                                                                                            |
+| Video Type                                        | ??                     | type                              | Yes       |                                                                                            |
+| Files (Videos, poster, subtitle)                  | 8564_, 8567_           | _files                            | Yes       | Video files will be decided with the webcast team.                                          |
+| Indico                                            | 8564_y, 8564_u, 970__a | related_links                     | No        | Indico link will be added as `Related Links`.                                               |
+| Copyright/ License                                | 540__/ 542__f          | license                           | No        | `URL` field will be added to the License, `statement` field will be added to the Copyright. |
+| Report Number                                     | 088__a                 | report_number                     | No        | Only old videos (digitised) do have it.                                                    |
+| Corporate Author                                  | 110__a                 | Not exist                         | Not exist | Not needed if hardcoded.                                                                  |
+| Note                                              | 500__a                 | Not exist                         | Not exist | Will be added as a hidden field `internal_note`.                                            |
+| Conference/ Presented at                          | 962__b                 | Not exist                         | Not exist | New fields for conference information.                                               |
+| Published in                                      | 773__                  | Not exist                         | Not exist | New fields for conference information.                                                |
+| Number of pages                                   | 300__a                 | Not exist                         | Not exist | Will be checked with the webcast team.                                                      |
+| Collaboration                                     | 710__g                 | Not exist                         | Not exist | Add a new role to Contributors.                                                             |
+| Related document                                  | 7870_                  | Not exist                         | Not exist | Will be `related works` as it’s done in CDS-RDM.                                           |
```

Comment:
Just a note: in the end in the data model this is called `related_identifiers`. Related works was the label chosen to display it in the UI